codegen/templates/endpoint.tmpl

```diff
@@ -60,6 +60,14 @@ func New{{$handlerName}}(
 ) *{{$handlerName}} {
 	return &{{$handlerName}}{
 		Clients: deps.Client,
+		Scope: gateway.AllHostScope.Tagged(map[string]string{
```

Comment:
Can we not allocate a scope in the template ? 

We can allocate one in runtime/ instead. we don't really need `thrift-method` or `workflow-type`

Reply:
Sure I can remove the extra tags: `module`, `module-type`, `thrift-method`, `workflow-type`.  They were there for completeness, the I can remove if not necessary.